static/app/views/issueDetails/traceTimeline/traceLink.tsx

```diff
@@ -51,9 +57,16 @@ export function TraceLink({event}: TraceLinkProps) {
     <StyledLink
       to={traceTarget}
       onClick={() => {
+        if (surface.includes('issue_details')) {
```

Comment:
could do something like

```
   trackAnalytics('quick_trace.trace_id.clicked', {
          organization,
          source: surface.includes('issue_details') ? 'issues' : surface,
   });
```
to remove the `if` block

Reply:
The old analytics event has a different name, which is used by people's redash/amplitude queries

Reply:
We might still want to emit the new style in either case. It'll be duplicated but this way we can eventually migrate to the new surface without losing data. Ultimately up to issues team though

Reply:
it looks like you duplicated the same analytic twice though